build.xml

```diff
@@ -165,13 +167,23 @@
         </condition>
         <echo if:set="saxon-available" message="saxon available: ${saxon-available}"/>
         <echo unless:set="saxon-available" message="saxon available: false"/>
+        <!-- check if stylesheets are available -->
+        <condition property="stylesheets-available">
+            <or>
```

Comment:
I guess, since we have only one conditional expression the "or" is not needed here